zerver/webhooks/taiga/view.py

```diff
@@ -250,7 +250,7 @@ def parse_message(message):
     return events
 
 def generate_content(data):
-    # type: (Mapping[str, Any]) -> str
+    # type: (Mapping[str, Any]) -> Union[str, HttpResponse]
```

Comment:
I think the type is correct, but this probably should be doing `raise JsonableError()`, not `return json_error`.